net-misc/youtube_dl/youtube_dl-2017.11.15.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A command-line program to download videos from various sites"
+DESCRIPTION="The program is usually invoked as youtube-dl followed by \
+options and the video URL. Run 'youtube-dl --help' and get a summary \
+of all options.
+Some YouTube videos are served using Adobe's proprietary RTMP protocol, which \
+imposes DRM restrictions and encrypts the connection. youtube-dl is not able \
+to download these videos by itself."
+HOMEPAGE="http://rg3.github.io/youtube-dl/"
+COPYRIGHT="2006-2017 Ricardo Garcia Gonzalez"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/rg3/youtube-dl/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="f43d65679bbff61a25bdb275f3d628596b1f8f73fd0bf38ee6b36c198b2c4b5d"
+SOURCE_DIR="youtube-dl-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	youtube_dl = $portVersion
+	cmd:youtube_dl = $portVersion
+	"
+REQUIRES="
+	haiku
+	cmd:python
+	cmd:ffmpeg
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:python
+	"
+BUILD_PREREQUIRES="
+	cmd:zip
+	"
+
+BUILD()
+{
+	zip -q youtube-dl youtube_dl/*.py youtube_dl/*/*.py
+	zip -q -j youtube-dl youtube_dl/__main__.py
+	echo '#!/bin/python' > youtube-dl
+	cat youtube-dl.zip >> youtube-dl
+	rm youtube-dl.zip
+	chmod a+x youtube-dl
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp -a youtube-dl $binDir
+}
+
+TEST()
+{
+	youtube-dl --version
```

Comment:
Nice, but youtube-dl provides plenty testcases in the test folder.
I fear, this TEST() section doesn't cover much functionality.